templates/python-scrapy/src/pipelines.py

```diff
@@ -1,18 +1,14 @@
 # Define your item pipelines here
 #
-# See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
+# See the Scrapy documentation: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
 
-from itemadapter import ItemAdapter
+import scrapy
 
-from scrapy import Item, Spider
+from .items import TitleItem
 
-from apify import Actor
 
+class TitleItemPipeline:
```

Comment:
This pipeline won't get applied unless you add it to settings

Reply:
I thought it could be just a placeholder. To let the users know where they can define their pipelines. However, we can just return the original Item and use it. Why not.